packages/website/src/components/Uploader/UploaderElement.js

```diff
@@ -0,0 +1,180 @@
+import * as React from "react";
+import classnames from "classnames";
+import { ArrowUpCircle, Error, CheckCircle, ProgressRound } from "../Icons";
+import bytes from "bytes";
+import { getReasonPhrase, StatusCodes } from "http-status-codes";
+import copy from "copy-text-to-clipboard";
+import path from "path-browserify";
+import { SkynetClient } from "skynet-js";
+import { useTimeoutFn } from "react-use";
+import ms from "ms";
+import Link from "../Link";
+
+const TOO_MANY_REQUESTS_RETRY = ms("5s"); // retry delay after "492: TOO_MANY_REQUESTS"
```

Comment:
do we need an exponential backoff? thinking about a situation where the retry effectively keeps pushing us over the rate limit...

Reply:
`nit` 429

Reply:
```suggestion
const TOO_MANY_REQUESTS_RETRY = ms("5s"); // retry delay after "429: TOO_MANY_REQUESTS"
```

Reply:
> do we need an exponential backoff? thinking about a situation where the retry effectively keeps pushing us over the rate limit...

added delay multiplication in https://github.com/SkynetLabs/skynet-webportal/pull/1029/commits/6bb7945fbfd81ba23affff680c09f7f65183491b